src/bin/languageserver/mod.rs

```diff
@@ -1677,6 +1758,89 @@ impl<'a> Builder<'a> {
                 });
 
             self.declarations.extend(decls);
+
+            //////////// Code objects defined within the contract ////////////
```

Comment:
```suggestion
            // Code objects defined within the contract
```
No ascii art please :)